plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/animators/PuckBearingAnimator.kt

```diff
@@ -5,7 +5,8 @@ import com.mapbox.maps.plugin.locationcomponent.OnIndicatorBearingChangedListene
 internal class PuckBearingAnimator(private val indicatorBearingChangedListener: OnIndicatorBearingChangedListener) : PuckAnimator<Double>(Evaluators.DOUBLE) {
 
   override fun updateLayer(fraction: Float, value: Double) {
-    locationRenderer?.setBearing(value)
+    val resolvedValue = 0.0.coerceAtLeast(value)
```

Comment:
why is that line needed?

Reply:
Good catch, I thought minus value is not valid for bearing. I will roll back this part.